### DISCUSSION TITLE
Migrating wdio v6 to v7, getting issue for Browserslist: caniuse-lite is outdated. Please run:   npx browserslist@latest --update-db

### DISCUSSION THREAD

**Discussion by sijites** [webdriverio/codemod#106] [Migration Issue]
After performing all the documented steps for v7 upgrade, getting below warn - 
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db. 
While performing npx browserslist@latest --update-db getting below error -
Could not resolve dependency:
npm WARN peer eslint@"^4.19.1 || ^5.3.0" from eslint-config-airbnb@17.1.1
npm WARN node_modules/eslint-config-airbnb
npm WARN   eslint-config-airbnb@"^17.1.0" from eslint-config-manulife@1.1.3
npm WARN   node_modules/eslint-config-manulife
npm WARN
npm WARN Conflicting peer dependency: eslint@5.16.0
npm WARN node_modules/eslint
npm WARN   peer eslint@"^4.19.1 || ^5.3.0" from eslint-config-airbnb@17.1.1
npm WARN   node_modules/eslint-config-airbnb
npm WARN     eslint-config-airbnb@"^17.1.0" from eslint-config-manulife@1.1.3
npm WARN     node_modules/eslint-config-manulife
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: eslint-config-airbnb-base@13.2.0
npm WARN Found: eslint@7.32.0
npm WARN node_modules/eslint

**Comment by christian-bromann** [comment]
@sijites this seems to be unrelated to WebdriverIO. I suggest follow the instructions mentioned in the error.
